benchmarks/GraphBLAS-sharp.Benchmarks/BenchmarksEWiseAdd.fs

```diff
@@ -70,6 +77,7 @@ type EWiseAddBenchmarks() =
         Cl.GetPlatformIDs &e
         |> Array.collect (fun platform -> Cl.GetDeviceIDs(platform, deviceType, &e))
         |> Seq.ofArray
+        |> Seq.distinctBy (fun device -> Cl.GetDeviceInfo(device, DeviceInfo.Name, &e).ToString())
```

Comment:
Devices are only "name-equal", not really (by-ref) equal?

Reply:
I think the way of filtering duplicates depends on the needs of the user. For example, I would like to consider that a single physical device with different driver versions are the same.